Classes/Utility/UserUtility.php

```diff
@@ -10,8 +10,7 @@
 use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
```

Comment:
The class `ExtensionManagementUtility` is now unused as well in this class, its use statement can be removed.